client/my-sites/media-library/content.jsx

```diff
@@ -24,7 +24,6 @@ import MediaLibrarySelectedData from 'components/data/media-library-selected-dat
 import MediaActions from 'lib/media/actions';
 import {
 	ValidationErrors as MediaValidationErrors,
-	MEDIA_IMAGE_PHOTON,
```

Comment:
If we're no longer using this constant for images, perhaps it's a good chance to cull it. For example, the [default thumbnailType](https://github.com/Automattic/wp-calypso/blob/047412b/client/my-sites/media-library/list-item-image.jsx#L30) for `<ListItemImage />` is still `MEDIA_IMAGE_PHOTON`. 

I'm not sure however how photon relates to videos, but it looks like we're passing down `thumbnailType` to all `<ListItem />` [children](https://github.com/Automattic/wp-calypso/blob/047412b/client/my-sites/media-library/list.jsx#L188), so on the face of it, it appears we can change the default type in all child components. 

It might be worth doing should, for any reason, we want to prevent `MEDIA_IMAGE_PHOTON` from being the default. For example, if the child components don't receive a valid `props.thumbnailType`.

If you do decide to change the default type, then we can also get rid of the corresponding tests.